targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF51822_UNIFIED/sdk_patch/sdk_config.h

```diff
@@ -6,8 +6,10 @@
 #define UART_DEFAULT_CONFIG_BAUDRATE UART0_CONFIG_BAUDRATE
 #define UART_DEFAULT_CONFIG_PARITY   UART0_CONFIG_PARITY
 #define UART_DEFAULT_CONFIG_HWFC     UART0_CONFIG_HWFC
-#define CTS_PIN_NUMBER               UART0_CONFIG_PSEL_CTS
-#define RTS_PIN_NUMBER               UART0_CONFIG_PSEL_RTS
+
+/* Use config at PinNames.h file in TARGET_XXXX directory */
```

Comment:
Hi @0xc0170 : My board do not use default config for RTS, CTS pin.

**Nordic nRF51822 based boards:**
- `nrf_drv_config.h`
#define UART0_CONFIG_PSEL_CTS     10
#define UART0_CONFIG_PSEL_RTS     8
- `TARGET_NRF51_DK/PinNames.h`
CTS_PIN_NUMBER  = p10,
RTS_PIN_NUMBER  = p8

**VBLUno51 board**
- `TARGET_VBLUNO51/PinNames.h`
CTS_PIN_NUMBER = p13,
RTS_PIN_NUMBER =  p12

I think that should use config for CTS, RTS pin from PinNames.h file.
This change allows each board to use different hardware configurations